tiyhw.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+###############################################################################
+#
+# This script is designed to make the homework git workflow easier for students
+# at The Iron Yard by automating the many git and hub commands frequently used.
+#
+# Place this file in ~/bin
+# 
+# Add the following to your .bash_profile (uncomment it first, duh)
+# 
+# # This loads the tiyhw file at the start of the terminal session.
+# eval ". ~/bin/tiyhw.sh"
+# # This sets an alias so the user can reload tiyhw.sh easily. 
+# alias tiy=". ~/bin/tiyhw.sh"
+#
+# Now in your terminal, type "tiy is_working" (without the "") to test. This 
+# script will install homebrew, git, and hub as needed, then print something
+# in your terminal if it is working correctly.
+#
+###############################################################################
+
+
+###############################################################################
+# 
+# This section of code declares the useful functions that will be found within.
+# 
+###############################################################################
+
+# This script requires homebrew, git, and hub. Calling system_check determines 
+# if the user has these and installs them if not.
+function system_check {
+  # Check for homebrew
+  if test ! $(which brew); then
+    echo "Installing homebrew..."
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  fi
+
+  # Check for hub
+  if test ! $(which hub); then
+    echo "Installing hub..."
+    brew istall hub
+  fi
+
+  # Check for git
+  if test ! $(which git); then
+    echo "Installing git..."
+    brew install git
+  fi
+}
+
+# This method creates a new github repository on your computer and on github.
+function new_repo {
+
+  # Intro and explain the script.
+  echo "This function creates a new repository on your computer and on GitHub."
+  echo ""
+
+  # Gather the name for the folder and the repo
+  read -p "Enter a new directory name: " folder
+  echo ""
+
+  # See if the user would like to exit this function.
+  if [ "$folder" != "end" ]; then 
```

Comment:
does this allow for someone to exit the prompt and whatnot? If so, `exit` might be more appropriate